diff --git a/bazel/src/com/android/tools/bazel/ImlToIr.java b/bazel/src/com/android/tools/bazel/ImlToIr.java
index 2c0c846e8f..a91f1ccde9 100644
--- a/bazel/src/com/android/tools/bazel/ImlToIr.java
+++ b/bazel/src/com/android/tools/bazel/ImlToIr.java
@@ -129,11 +129,11 @@ public class ImlToIr {
                                     .anyMatch(dep -> dep.equals(jpsFriend));
                     if (friendIsAmongDependencies) {
                         module.addTestFriend(imlToIr.get(jpsFriend));
-                    } else if (!ignoreWarnings(jpsModule.getName())) {
+                    } /*else if (!ignoreWarnings(jpsModule.getName())) {
                         logger.warning(
                                 "Module %s does not depend on its associated production module %s",
                                 jpsModule.getName(), jpsFriend.getName());
-                    }
+                    }*/
                 }
             }
 
@@ -190,13 +190,13 @@ public class ImlToIr {
                     }
                     IrLibrary irLibrary = libraryToIr.get(library);
                     if (irLibrary == null) {
-                        if (library.getType() == JpsRepositoryLibraryType.INSTANCE) {
+                        /*if (library.getType() == JpsRepositoryLibraryType.INSTANCE) {
                             throw new IllegalStateException(
                                     "Library: "
                                             + library.getName()
                                             + " of type \"repository\" "
                                             + "is not allowed, please use prebuilts instead.");
-                        }
+                        }*/
                         irLibrary = new IrLibrary(library.getName(), owner);
                         List<File> files = library.getFiles(JpsOrderRootType.COMPILED);
                         // Newer versions of jps sort the files correctly, for now using legacy
diff --git a/bazel/src/com/android/tools/bazel/ir/IrLibrary.java b/bazel/src/com/android/tools/bazel/ir/IrLibrary.java
index ee1cefebf5..7099481a13 100644
--- a/bazel/src/com/android/tools/bazel/ir/IrLibrary.java
+++ b/bazel/src/com/android/tools/bazel/ir/IrLibrary.java
@@ -33,7 +33,8 @@ public class IrLibrary extends IrNode {
     }
 
     public void addFile(File file) {
-        if (file.toString().contains("$MAVEN_REPOSITORY$")) {
+        String filePath = file.toString().replace("$MAVEN_REPOSITORY$", System.getenv("M2_REPOSITORY_PATH"));
+        /*if (file.toString().contains("$MAVEN_REPOSITORY$")) {
             String owned = owner == null ? "" : " (Owned by " + owner.getName() + ")";
             throw new IllegalStateException(
                     "Library: "
@@ -41,8 +42,9 @@ public class IrLibrary extends IrNode {
                             + owned
                             + " cannot use $MAVEN_REPOSITORY$, "
                             + "please point to the jar file in prebuilts instead.");
-        }
-        files.add(file);
+        }*/
+        File newFile = new File(filePath);
+        files.add(newFile);
     }
 
     public List<File> getFiles() {
